MAUI/TreeView/right-to-left.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Right To Left in .NET MAUI TreeView Control | Syncfusion
+description: Learn here all about Right To Left support in Syncfusion .NET MAUI TreeView (SfTreeView) Control and more.
```

Comment:
description: Learn here about Right To Left support in Syncfusion .NET MAUI TreeView (SfTreeView) Control and more.

Reply:
Changed the description with the above content.